docs/integrations/sources/postgres.md

```diff
@@ -105,3 +106,7 @@ ALTER DEFAULT PRIVILEGES IN SCHEMA <schema_name> GRANT SELECT ON TABLES TO airby
 
 Your database user should now be ready for use with Airbyte.
 
+#### Set the default Postgres client encoding to UTF8
```

Comment:
It is weird that they ahve to modify the server config so that the client can deal with UTF-8 encoded data. Is it something that can be handled at the integration level?

Reply:
They are able to query that data without having to change the server so this solution seems incorect.

Reply:
We may be able to fix this with an env variable in the tap container `PGCLIENTENCODING=UTF8` which looks like it's respected by psycopg2.

Reply:
Great call, fixed in #382